lib/src/view/game/analysis_screen.dart

```diff
@@ -0,0 +1,944 @@
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              EngineGauge(
+                displayMode: EngineGaugeDisplayMode.horizontal,
+                params: EngineGaugeParams(
+                  evaluationContext: analysisState.evaluationContext,
+                  position: analysisState.position,
+                  savedEval: analysisState.currentNode.eval,
+                ),
+              ),
+              _CevalLines(ctrlProvider, isTablet: false),
+            ],
+          )
+        : const SizedBox.shrink();
+  }
+}
+
+class _EngineGaugeVertical extends ConsumerWidget {
+  const _EngineGaugeVertical(this.ctrlProvider);
+
+  final AnalysisCtrlProvider ctrlProvider;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final analysisState = ref.watch(ctrlProvider);
+
+    return analysisState.isEngineEnabled
+        ? EngineGauge(
+            displayMode: EngineGaugeDisplayMode.vertical,
+            params: EngineGaugeParams(
+              evaluationContext: analysisState.evaluationContext,
+              position: analysisState.position,
+              savedEval: analysisState.currentNode.eval,
+            ),
+          )
+        : kEmptyWidget;
+  }
+}
+
+class _CevalLines extends ConsumerWidget {
+  const _CevalLines(this.ctrlProvider, {required this.isTablet});
+  final AnalysisCtrlProvider ctrlProvider;
+  final bool isTablet;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final analysisState = ref.watch(ctrlProvider);
+    final ceval =
+        ref.watch(engineEvaluationProvider(analysisState.evaluationContext));
+
+    final content = ceval != null
+        ? ceval.pvs
+            .map(
+              (pv) => _CevalLine(
+                ctrlProvider,
+                pv.sanMoves(ceval.currentPosition),
+                analysisState.currentNode.ply,
+                pv,
+              ),
+            )
+            .toList()
+        : (analysisState.currentNode.eval != null
+            ? analysisState.currentNode.eval!.pvs
+                .take(analysisState.numCevalLines)
+                .map(
+                  (pv) => _CevalLine(
+                    ctrlProvider,
+                    pv.sanMoves(analysisState.currentNode.position),
+                    analysisState.currentNode.ply,
+                    pv,
+                  ),
+                )
+                .toList()
+            : null);
+
+    return content != null
+        ? Padding(
+            padding: EdgeInsets.symmetric(
+              vertical: isTablet ? kTabletPadding : 4.0,
+              horizontal: isTablet ? kTabletPadding : 4.0,
+            ),
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: content,
+            ),
+          )
+        : kEmptyWidget;
+  }
+}
+
+class _CevalLine extends ConsumerWidget {
```

Comment:
In lila code it's called `Ceval` but here I never used that name so far... to avoid confusion I'd not use it and rename to `EngineLine`